core/src/main/java/com/scalar/db/storage/common/TableMetadataManager.java

```diff
@@ -37,4 +38,12 @@
    * @param metadata a table metadata
    */
   void addTableMetadata(String namespace, String table, TableMetadata metadata);
+
+  /**
+   * Return the names of the table belonging to the given namespace
```

Comment:
For consistency.
```suggestion
   * Returns the names of the table belonging to the given namespace
```
Ditto for `Delete` and `Add`.